app/mailers/nsm/submission_mailer.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+module Nsm
+  class SubmissionMailer < GovukNotifyRails::Mailer
+    def notify(claim)
+      @claim = claim
+      set_template('0403454c-47a5-4540-804c-cb614e77dc22')
+      set_personalisation(
+        LAA_case_reference: case_reference,
+        UFN: unique_file_number,
+        main_defendant_name: defendant_name,
+        defendant_reference: defendant_reference_string,
+        claim_total: claim_total,
+        date: submission_date,
+        feedback_url: feedback_url
+      )
+      mail(to: email_recipient)
+    end
+
+    private
+
+    def email_recipient
+      @claim.submitter.email
+    end
+
+    def case_reference
+      @claim.laa_reference
+    end
+
+    def unique_file_number
+      @claim.ufn
+    end
+
+    def main_defendant
+      @claim.defendants.find_by(main: true)
```

Comment:
So this one is my fault - I was late to spot that there's already a `main_defendant` method on the Claim object you can use, but that's probably preferable, as otherwise methods like `defendant_name` will all cause a SQL query to occur.